Xbim.Properties/Xbim.Properties.csproj

```diff
@@ -1,114 +1,115 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-	<PropertyGroup>
-		<OutputType>Library</OutputType>
-		<RestorePackages>true</RestorePackages>
-		<TargetFrameworks>netstandard2.0;netstandard2.1</TargetFrameworks>
-		<PackageLicenseExpression>CDDL-1.0</PackageLicenseExpression>
-		<PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
-		<Authors>xBIM Team</Authors>
-		<PackageProjectUrl>https://github.com/xBimTeam/XbimPropertySets</PackageProjectUrl>
-		<PackageIcon>xbim-toolkit-icon.png</PackageIcon>
-		<RepositoryUrl>https://github.com/xBimTeam/XbimPropertySets.git</RepositoryUrl>
-		<RepositoryType>git</RepositoryType>
-		<PackageTags>BIM, IFC, BuildingSmart, PropertySets, Property, IFD, bSDD</PackageTags>
-		<PackageReleaseNotes>Now supporting IFC4x3 schema</PackageReleaseNotes>
-		<AssemblyVersion>6.0.0</AssemblyVersion>
-		<FileVersion>6.0.0</FileVersion>
-    <Version>6.0.0</Version>
-		<SignAssembly>true</SignAssembly>
-		<AssemblyOriginatorKeyFile>XbimOpenSourceKeyFile.snk</AssemblyOriginatorKeyFile>
-		<Title>xbim.PropertySets</Title>
-		<Description>Programmatic access to BuildingSmart Properties and Psets for IFC schemas, and related Data Dictionaries</Description>
-	</PropertyGroup>
-	<PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|netstandard2.0|AnyCPU'">
-	  <DefineConstants>DEBUG;TRACE</DefineConstants>
-	</PropertyGroup>
+  <PropertyGroup>
+    <OutputType>Library</OutputType>
+    <RestorePackages>true</RestorePackages>
+    <TargetFrameworks>net8.0</TargetFrameworks>
```

Comment:
I think this should be
<TargetFrameworks>netstandard2.0;netstandard2.1;net8.0</TargetFrameworks>